com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -609,63 +598,63 @@ private bool ApplyTransformToNetworkStateWithInfo(ref NetworkTransformState netw
                 isDirty = true;
             }
 
-            if (SyncPositionX && Mathf.Abs(networkState.PositionX - position.x) >= PositionThreshold || networkState.IsTeleportingNextFrame)
+            if (SyncPositionX && (Mathf.Abs(networkState.PositionX - position.x) >= PositionThreshold || networkState.IsTeleportingNextFrame))
```

Comment:
This bug I found while debugging parent-child scale issues when the NetworkObject also had a NetworkTransform.  I had turned the NetworkTransform scale synchronization flags (X, Y,Z) off but it seemed to keep updating them...which made me realize the initial state after 1 or 2 network ticks is a "teleport state"...which the previous logic (without the braces) would apply all axis whether marked to be synchronized or not.